.tool-versions

```diff
@@ -1,2 +1,2 @@
-nodejs 18.20.4 18.20.3
+nodejs 18.20.6
```

Comment:
Not related to LCOV, but breaks CI. So I agree that it's good to include this asap.